extensions/ql-vscode/src/remote-queries/gh-api/gh-api-client.ts

```diff
@@ -0,0 +1,86 @@
+import { Credentials } from '../../authentication';
+import { OctokitResponse } from '@octokit/types/dist-types';
+import { VariantAnalysisSubmission } from '../shared/variant-analysis';
+import {
+  VariantAnalysis,
+  VariantAnalysisQueryLanguage,
+  VariantAnalysisRepoTask,
+  VariantAnalysisSubmissionRequest
+} from './variant-analysis';
+
+export async function submitVariantAnalysis(
+  credentials: Credentials,
+  submissionDetails: VariantAnalysisSubmission
+): Promise<VariantAnalysis> {
+  const octokit = await credentials.getOctokit();
+
+  const { actionRepoRef, query, databases, controllerRepoId } = submissionDetails;
+
+  const data: VariantAnalysisSubmissionRequest = {
+    action_repo_ref: actionRepoRef,
+    language: query.language as VariantAnalysisQueryLanguage,
+    query_pack: query.pack,
+    repositories: databases.repositories,
+    repository_lists: databases.repositoryLists,
+    repository_owners: databases.repositoryOwners,
+  };
+
+  const response: OctokitResponse<VariantAnalysis, number> = await octokit.request(
```

Comment:
`number` is already the default of the second parameter to `OctokitResponse`, so this should be sufficient:

```suggestion
  const response: OctokitResponse<VariantAnalysis> = await octokit.request(
```